src/libraries/System.Security.Cryptography.X509Certificates/tests/CertTests.cs

```diff
@@ -122,22 +122,6 @@ public void TestVerify()
 
                 Assert.True(success, "MicrosoftDotComIssuerBytes");
             }
-
-            // High Sierra fails to build a chain for a self-signed certificate with revocation enabled.
-            // https://github.com/dotnet/runtime/issues/22625
-            if (!PlatformDetection.IsMacOsHighSierraOrHigher)
```

Comment:
Not sure the best way to handle this. The code is effectively dead, maybe it would be better to move it to a new test, put an `ActiveIssue` with the above GitHub link?

Reply:
Sounds reasonable.